requirements.txt

```diff
@@ -1,7 +1,7 @@
-lit==15.0.6
+lit~=15.0
 # numpy 1.24 deprecates np.object, np.bool, np.float, np.complex, np.str,
 # and np.int which are used heavily in onnx-mlir.
-numpy>=1.19.5, <=1.23.5
-protobuf==3.20.3
-pytest>=4.6.9, <=7.2.0
-pytest-xdist>=1.31.0, <=2.5
+numpy>=1.17.4, <=1.23.5
+protobuf~=3.20
```

Comment:
I am not sure if we want to do this for protobuf specifically because the pipelines checkout and build a specific version which is 3.20.3. I don't think it's blocking though since it will most likely work fine even if there's a mismatch.